docs/pages/management/security/proxy-protocol.mdx

```diff
@@ -1,88 +1,134 @@
 ---
-title: PROXY protocol
-description: How to securely configure PROXY protocol usage with Teleport
+title: Run Teleport Behind a Layer 4 Load Balancer
+description: Describes how to securely configure Teleport to use the PROXY protocol so you can run the Auth Service and Proxy Service behind a layer 4 load balancer.
```

Comment:
Don't we just use PROXY protocol to preserve the original client IP? If so, it's not necessarily required to run behind a L4 LB.